tests/system_tests/__init__.py

```diff
@@ -89,7 +89,7 @@ def __getattr__(self, name):
 
         def invocation(url, *args, **kwargs):
             return getattr(requests, name)(
-                    "{}:{}{}".format(self.host, self.port, url),
+                    ("{}:{}" + url).format(self.host, self.port, hostname=self.switch.hostname),
```

Comment:
Putting the url in the string BEFORE the format lets users benefit from {hostname} variable